src/ensembl/src/ens-object/ensObjectTypes.ts

```diff
@@ -24,6 +24,7 @@ export type EnsObjectTrack = {
   ensembl_object_id?: string;
   support_level?: string | null;
   track_id: string;
+  description: string;
```

Comment:
Wondering about the types of EnsObjectTrack:
1) is it really so that most of the fields can be absent? Or is it simply that their value can be null?
2) have you agreed with the backend that the description field will always be present and non-null?

Reply:
Ok, it looks like various fields indeed can be absent.
Still wondering whether the description field will always be present though.

Reply:
At the moment the description field is present in all the objects we currently support. 
@sanjay-boddu could you confirm if a value will always be there for description, please?

Reply:
@azangru we are not very sure whether the middleware will return null or a default value for missing descriptions at this point. Should we leave this for now and come back to it later?

Reply:
ok. If we are that unsure, I think it will be safer to describe this field as `string | null`.

Reply:
That's done.